var/spack/repos/builtin/packages/express/package.py

```diff
@@ -53,10 +53,8 @@ def patch(self):
                                 'bamtools'].prefix.include.bamtools.api)
             edit = FileFilter('CMakeLists.txt')
             # really ugly way to escape the $ in CMakeLists.txt
```

Comment:
You can get rid of this comment now. In fact, if you use `string=True`, you won't need to escape it at all.